tests/percy/lib/util.js

```diff
@@ -15,6 +15,16 @@ exports.unmasquerade = async function(page, HOME_PAGE) {
   await page.goto(`${HOME_PAGE}${link}`);
 }
 
+exports.removeAlert = async function(url, HOME_PAGE, page) {
+  await page.goto(`${HOME_PAGE}/${url}`, { waitUntil: "load" });
+    // Remove sitewide alert
+      await page.evaluate(() =>
```

Comment:
I wonder if an exception will be thrown when there is no alert to begin with. Could you test by unpublishing all alerts? If there is an exception, can add a try/catch block around it. In this case can even ignore the exception since no alert is not an error condition.

Reply:
Tested this and the test still pass if there isn't an alert.